README.md

```diff
@@ -1,6 +1,9 @@
 # Neat URL
 
-Neat URL cleans URLs, removing parameters such as Google Analytics' utm parameters.
+Neat URL is a browser extension for [Firefox][] and [Chrome][], that cleans URLs, removing parameters such as Google Analytics' utm parameters.
```

Comment:
`Neat URL is a browser extension for that cleans URLs, removing parameters such as Google Analytics' utm parameters. Use with [Firefox][] and [Chrome][]`

I think the way you have it is a little hard to read.

Reply:
At the very least the comma after [Chrome][] in the suggestion needs to be removed unless it's reworded. @kd2718's suggestion is terse but grammatically correct. :+1: